<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_subrecv</class>
 <widget class="QWidget" name="page_subrecv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="Input" name="input_recv">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlyt">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmb_filter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>StartsWith</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EndsWith</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Contains</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Not StartsWith</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Not EndsWith</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Not Contains</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_filter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Input</class>
   <extends>QTextEdit</extends>
   <header>input.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
